__tests__/app.test.js

```diff
@@ -243,3 +243,22 @@ describe("DELETE /api/articles/:article_id", () => {
     expect(response.body.msg).toBe("Bad Request");
   });
 });
+
+describe("GET /api/users", () => {
+  it("200: responds with all of the users", async () => {
+    const response = await request(app).get("/api/users");
+    expect(response.status).toBe(200);
+    expect(response.body.users.length).toBe(4);
+    response.body.users.map((user) => {
+      expect(user).toHaveProperty("username");
+      expect(user).toHaveProperty("avatar_url");
+      expect(user).toHaveProperty("name");
```

Comment:
We should be asserting against the typeof the values as well here, not just the keys.